### PR TITLE
OSDOCS#5162:Web console dynamic plugin updates RN

### PR DESCRIPTION
**Pull request by opayne1**
OSDOCS#5162:Web console dynamic plugin updates RN

Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-5162

Link to docs preview:
https://66230--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#web-console-dynamic-plugin-enhancements

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -214,6 +214,13 @@ With this release, there are several updates to the *Administrator* perspective
 * Provide direct feedback about features and report a bug by clicking the *Help* button on the toolbar and clicking *Share Feedback* from the drop-down list.
 * Display and hide tooltips in the YAML editor. The tooltips will persist so you don't have to change it every time you navigate to the page.
 
+[id="web-console-dynamic-plugin-enhancements"]
+===== Dynamic plugin enhancements
+
+With this update, you can add custom metric dashboards and extend the cluster's *Overview* page with the `QueryBowser` extension. Additional extension points have been added which allow you to add different types of modals, set the active namespace, provie custom error pages, and set proxy timeouts for your dynamic plugin.
```
Comment (spadgett) [inline release_notes/ocp-4-14-release-notes.adoc:220]: Consider linking to the QueryBrowser API doc.

Comment (spadgett) [inline release_notes/ocp-4-14-release-notes.adoc:220]: @jhadvig @vojtechszocs @TheRealJon FYI, let us know if this looks good.

**Review by spadgett — COMMENTED** [review]
Thanks, @opayne1 !

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -214,6 +214,14 @@ With this release, there are several updates to the *Administrator* perspective
...
```
[the remark this answers is not in the document]
Comment (mletalie) [reply release_notes/ocp-4-14-release-notes.adoc:220]: Assuming that should be "provide"? Possibly "provision"?

**Comment by mletalie** [comment]
LGTM!

**Comment by yanpzhan** [comment]
LGTM

**Pull request merged**